qemu/tests/cluster_size_check.py

```diff
@@ -53,22 +53,20 @@ def check_cluster_size(parttern, expect, csize_set):
 
         if status_error:
             if result.exit_status == 0:
-                logging.error("Return success with invalid Cluster size: %s"
+                logging.error("Create image sucessfully with invalid size: %s"
                               % csize_set)
-                logging.error("%s.\n" % result)
                 cfail += 1
+                fail_log += "Succeed in creating image unexpectedly.\n"
         else:
             output = image.info()
             error.context("Check the cluster size from output", logging.info)
             cluster_size = re.findall(parttern, output)
             if cluster_size:
                 if cluster_size[0] != expect:
```

Comment:
Could you please also help update the type of cluster_size?
14:59:45 ERROR| Cluster size ['4096'] is not expected value 4096

Others are look good to me.
Thanks.